tests/services/apiService.test.js

```diff
@@ -40,3 +41,87 @@ describe("fetchQuestCategoryData function", () => {
     expect(result).toEqual(mockResponse);
   });
 });
+
+describe("getDeployedTimeByAddress function", () => {
+  beforeEach(() => {
+    fetch.mockClear();
+  });
+
+  it("should fetch and return data for a valid address or domain", async () => {
+    const mockData = {
+      timestamp: 9843327487
+    };
+    fetch.mockResolvedValueOnce({
+      json: () => Promise.resolve(mockData),
+    });
+
+    const result = await getDeployedTimeByAddress("address-or-domain");
```

Comment:
this is not a valid address. can you please test the endpoint response in all scenarios and then handle the test cases

Reply:
this will not return any response when you call the api